.releaserc.json

```diff
@@ -9,7 +9,12 @@
 				"changelogFile": "CHANGELOG.md"
 			}
 		],
-		"@semantic-release/github",
+		[
+			"@semantic-release/github",
+			{
+				"failTitle": false
```

Comment:
🤔 this is going to disable issues when _any_ release fails, right?

I'm thinking that issue is a good one normally. If releases start to fail -e.g. from an npm token getting disabled- then the repo should know about it.

What #33 is really asking is that the _initial commit_ for a repo be ignored.

Reply:
Thanks for your kind review.

My new proposed solution is to add an if condition based on the commit message to skip the release action, what do you think?

![image](https://user-images.githubusercontent.com/3297859/194789896-bfded908-a348-40ce-b324-b8df61233012.png)

Reply:
Yeah that seems very reasonable! +1 😄

Reply:
Although, we wouldn't want it to be _any_ commit that _happens_ to contain _"Initial commit"_. Maybe an equality check, rather than a contains?

Reply:
Sure